src/syntax/Parser.js

```diff
@@ -2247,7 +2247,8 @@ export class Parser {
     let name = this.parsePropertyName_();
 
     if (this.options_.propertyMethods && this.peek_(OPEN_PAREN)) {
-      let fs = this.pushFunctionState_(FUNCTION_STATE_METHOD);
+      let fs = this.pushFunctionState_(FUNCTION_STATE_METHOD |
+          FUNCTION_STATE_DERIVED_CONSTRUCTOR);
```

Comment:
This doesn't look right. I assume this is a left over from when you were trying to make the placeholder parser to work.